packages/api/core/test/slow/api_spec_slow.ts

```diff
@@ -165,6 +165,52 @@ describe(`electron-forge API (with installer=${nodeInstaller})`, () => {
     });
   });
 
+  describe('init (with typescript templater)', () => {
+    before(ensureTestDirIsNonexistent);
+
+    it('should succeed in initializing the webpack template', async () => {
+      await forge.init({
+        dir,
+        template: 'typescript',
+      });
+    });
+
+    it('should fail in initializing an already initialized directory', async () => {
+      await expect(forge.init({
+        dir,
+        template: 'typescript',
+      })).to.eventually.be.rejected;
+    });
+
+    it('should initialize an already initialized directory when forced to', async () => {
+      await forge.init({
+        dir,
+        force: true,
+        template: 'typescript',
+      });
+    });
```

Comment:
We probably don't need to run these tests again with a different built-in template. `init` takes a bit to run (mostly because of installing dependencies) and I'd like to avoid adding more time to CI when possible.